airbyte-integrations/connectors/source-recharge/unit_tests/integration/streams/test_shop.py

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+import json
+from unittest import TestCase
+
+import freezegun
+from airbyte_cdk.test.mock_http import HttpMocker, HttpResponse
+from airbyte_cdk.test.mock_http.response_builder import find_template
+
+from ..config import NOW
+from ..request_builder import get_stream_request
+from ..utils import config, read_full_refresh
+
+_STREAM_NAME = "shop"
+
+
+@freezegun.freeze_time(NOW.isoformat())
+class TestFullRefresh(TestCase):
+    @HttpMocker()
+    def test_given_one_page_when_read_then_return_records(self, http_mocker: HttpMocker) -> None:
```

Comment:
nit: Is it possible to have more than one page. Should this test name just be `test_when_read_return_single_record`?

Reply:
Fixed